### ISSUE TITLE
Crashes when seedRatioLimit is not an int

### ISSUE THREAD

**Issue by rubenv** [metalmatze/transmission-exporter#1]
Session.seedRatioLimit should probably be a float64, since this value isn't always an integer value

**Comment by captn3m0** [comment]
Facing the same issue.

**Comment by metalmatze** [comment]
Is someone creating a PR?
Otherwise I need to find time. 😐

**Comment by lord2800** [comment]
The same kind of thing happens when you have too much free space:

```
2018/01/08 20:07:15 failed to get session: json: cannot unmarshal number 3663358976000 into Go struct field Session.download-dir-free-space of type int
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x4 pc=0x245910]
goroutine 31 [running]:
main.(*SessionCollector).Collect(0x107a6400, 0x107a6fc0)
        src/github.com/metalmatze/transmission-exporter/cmd/transmission-exporter/session_collector.go:135 +0x84
github.com/metalmatze/transmission-exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0x107a4d30, 0x107a6fc0, 0x402098, 0x107a6400)
        src/github.com/metalmatze/transmission-exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:382 +0x50
created by github.com/metalmatze/transmission-exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather
        src/github.com/metalmatze/transmission-exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:383 +0x204
```

**Comment by captn3m0** [comment]
@metalmatze I filed a PR: #2 

Have tested it on native-transmission only, didn't manage to do a Docker build.

**Comment by metalmatze** [comment]
I've merged the PR and released 0.2.1:
https://github.com/metalmatze/transmission-exporter/releases/tag/0.2.1
https://hub.docker.com/r/metalmatze/transmission-exporter/tags/

Thanks for helping out!
